pom.xml

```diff
@@ -43,6 +42,11 @@
       <artifactId>xercesImpl</artifactId>
       <version>2.11.0</version>
     </dependency>
+    <dependency><!-- 2.4 is required. Avoid transitive import derived issues -->
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.4</version>
```

Comment:
Where is this library required? We need to be sure not to break [JENKINS-26923](https://issues.jenkins-ci.org/browse/JENKINS-26923) again.